ddpui/api/public_api.py

```
@@ -1222,21 +1194,27 @@ def get_public_report_chart_data(request, token: str):
-def get_public_report_table_data(request, token: str, page: int = 0, limit: int = 100):
+def get_public_report_table_data(
+    request, token: str, chart_id: int, page: int = 0, limit: int = 100
+):
     """Get table chart data for a public report"""
     try:
-        snapshot = _get_public_report_snapshot(token, request=request)
-
-        org_warehouse = OrgWarehouse.objects.filter(org=snapshot.org).first()
-        if not org_warehouse:
-            raise Exception("No warehouse configured for organization")
+        snapshot, chart_config, org_warehouse = _get_frozen_chart_for_public_report(
```

Ruff (0.15.9)

[warning] Unpacked variable `snapshot` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)